clients/drcachesim/scheduler/scheduler.cpp

```diff
@@ -1321,6 +1321,10 @@ scheduler_tmpl_t<RecordType, ReaderType>::time_tree_lookup(
     uint64_t lower_ord = it->second;
```

Comment:
s/skip-to-timestap/skip-to-timestamp/ in the PR description.

Reply:
Done.